### DISCUSSION TITLE
Where is ca.pem?

### DISCUSSION THREAD

**Discussion by danielgerlag** [dotnet/dotnet-operator-sdk#481] [Q&A]
I tried to run/deploy the demo that is generated from the template, but I seem to be missing a ca.pem file?
Where do I ge this?

**Comment by buehler** [comment]
Do you use a webhook? The certificates are only configured when you actively use a migration/validation webhook

**Reply by danielgerlag** [reply]
Yes, I used the template from https://buehler.github.io/dotnet-operator-sdk/docs/templates.html
It comes out of the box with a webhook, I simply generated the template and tried to deploy it.

**Reply by buehler** [reply]
Actually, the ca.pem should be generated while building
